src/3p-frame.js

```diff
@@ -17,14 +17,15 @@
 
 import {assert} from './asserts';
 import {getLengthNumeral} from '../src/layout';
+import {getService} from './service';
 import {documentInfoFor} from './document-info';
 import {getMode} from './mode';
 import {dashToCamelCase} from './string';
-import {parseUrl} from './url';
+import {parseUrl, assertHttpsUrl} from './url';
 
 
 /** @type {!Object<string,number>} Number of 3p frames on the for that type. */
-var count = {};
+const count = {};
```

Comment:
We'll need to discuss that rule at some point. I think it is weird and ugly here.